x/posts/internal/types/post_test.go

```diff
@@ -54,7 +88,7 @@ func TestPost_Validate(t *testing.T) {
 	}{
 		{
 			post:     types.Post{PostID: types.PostID(0)},
-			expError: "invalid post id: 0",
+			expError: "invalid post newID: 0",
```

Comment:
I think this is a refactoring error

```suggestion
			expError: "invalid post id: 0",
```